exporter/elasticexporter/README.md

```diff
@@ -1,6 +1,13 @@
 # Elastic native support for OpenTelemetry
+# Deprecated Elastic Exporter
 
-ℹ️ Since version 7.13, Elastic supports native OTLP ingestion. This means you can use an [OTLP exporter](https://github.com/open-telemetry/opentelemetry-collector/tree/main/exporter/otlpexporter) to send data to Elastic, instead of the OpenTelemetry Collector Exporter for Elastic. **The `otlp` exporter is the recommended way to integrate the OpenTelemetry Collector to Elastic.**
+| Status                   |                 |
+| ------------------------ |-----------------|
+| Stability                | [deprecated]    |
+| Supported pipeline types | trace, metrics  |
+| Distributions            | [contrib]       |
+
+ℹ️ This exporter has been deprecated due to elastic supports native OTLP ingestion since version 7.13. This means you can use an [OTLP exporter](https://github.com/open-telemetry/opentelemetry-collector/tree/main/exporter/otlpexporter) to send data to Elastic, instead of the OpenTelemetry Collector Exporter for Elastic. **The `otlp` exporter is the recommended way to integrate the OpenTelemetry Collector to Elastic.**
```

Comment:
```suggestion
ℹ️ This exporter has been deprecated due to Elastic APM Server supporting native OTLP ingestion since version 7.13. This means you can use an [OTLP exporter](https://github.com/open-telemetry/opentelemetry-collector/tree/main/exporter/otlpexporter) to send data to Elastic APM Server, instead of the OpenTelemetry Collector Exporter for Elastic. **The `otlp` exporter is the recommended way to integrate the OpenTelemetry Collector to Elastic.**
```